utilities_common/portstat.py

```diff
@@ -91,7 +100,11 @@
         41: ['SAI_PORT_STAT_IP_IN_RECEIVES'],
         42: ['SAI_PORT_STAT_IF_IN_FEC_CORRECTABLE_FRAMES'],
         43: ['SAI_PORT_STAT_IF_IN_FEC_NOT_CORRECTABLE_FRAMES'],
-        44: ['SAI_PORT_STAT_IF_IN_FEC_SYMBOL_ERRORS']
+        44: ['SAI_PORT_STAT_IF_IN_FEC_SYMBOL_ERRORS'],
```

Comment:
If WRED profile action is 'ecn_marking', does this port counter(SAI_PORT_STAT_GREEN_WRED_DROPPED_PACKETS) show ECN marked packets or this is used only for WRED drop?